ImagePickerModule.java

```diff
@@ -196,7 +199,9 @@ public void onCustomButton(@NonNull final ImagePickerModule module,
 
   public void doOnCancel()
   {
-    responseHelper.invokeCancel(callback);
+    if (this.callback != null) {
```

Comment:
The `this.` is technically unnecessary here, but the longer I've been away from coding Java, the more I think it's dumb that it isn't necessary